src/DefaultBuilder/src/GenericHostBuilderExtensions.cs

```diff
@@ -13,7 +13,8 @@ namespace Microsoft.Extensions.Hosting
     public static class GenericHostBuilderExtensions
     {
         /// <summary>
-        /// Configures a <see cref="IHostBuilder" /> with defaults for hosting a web app.
+        /// Configures a <see cref="IHostBuilder" /> with defaults for hosting a web app. This will overwrite
+        /// previously configured values and is intended to be called before additional configuration calls.
```

Comment:
This is pretty vague about what might be overwritten. Is the concern about services, IConfiguration, or something else?

Reply:
I think it's okay to be a bit vague, yes to all of the above. If this is called after `.UseEnvironment` for example it will overwrite the desired environment intended for use. The same is true for `.UseContentRoot` and services - as far as I understand it.